KlingonAssistant/data/mem-12-q.xml

```diff
@@ -1243,12 +1243,12 @@ The subtitle was "What's she doing?" but a more literal translation is "Why is s
       <column name="entry_name">qavan!</column>
       <column name="part_of_speech">sen:bc</column>
       <column name="definition">I salute you!</column>
-      <column name="definition_de">Ich grüße dir!</column>
+      <column name="definition_de">Ich salutiere dir!</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also"></column>
-      <column name="notes">This is used when addressing a single person. To salute multiple people, say {Savan!:sen}</column>
-      <column name="notes_de"></column>
+      <column name="notes">This is used when addressing a single person. To salute multiple people, say {Savan!:sen}.</column>
```

Comment:
There shouldn't be a period. The sentence already ends in an exclamation mark.

Reply:
Okay. Thanks fo telling me. I thought the exclamation mark was part of the definition.